AIDevGallery.SourceGenerator/SamplesSourceGenerator.cs

```diff
@@ -66,7 +66,15 @@ private void ExecuteSharedCodeEnumGeneration(SourceProductionContext context, Im
         foreach (var filePath in filePaths)
         {
             var fileName = Path.GetFileNameWithoutExtension(filePath);
-            if (File.Exists(Path.ChangeExtension(filePath, ".xaml")))
+            var extension = Path.GetExtension(filePath);
+            var filePathWithoutExtension = filePath.Substring(0, filePath.Length - extension.Length);
+            if (fileName.EndsWith(".xaml", StringComparison.InvariantCultureIgnoreCase) && File.Exists(filePathWithoutExtension))
+            {
+                fileName = Path.GetFileNameWithoutExtension(filePathWithoutExtension);
+                sourceBuilder.AppendLine($"        {fileName}Cs,");
+                sourceBuilder.AppendLine($"        {fileName}Xaml,");
+            }
+            else if (File.Exists(Path.ChangeExtension(filePath, ".xaml")))
             {
                 sourceBuilder.AppendLine($"        {fileName}Cs,");
```

Comment:
Is this ever being called now? I mean for xaml files and not xaml.cs files? If its not, we could remove this.

Reply:
Yes it is.